### ISSUE TITLE
Conflicts between kindred and lxml

### ISSUE THREAD

**Issue by NeolithEra** [jakelever/kindred#16]
Hi, users are unable to run **_kindred_** due to dependency conflict with _**lxml**_ package. 
As shown in the following full dependency graph of  kindred , kindred  requires _**lxml**_(the latest version)，while bioc==1.2.3 requires _**lxml==4.2.1**_.
According to pip’s “first found wins” installation strategy, _**lxml 4.3.4**_ is the actually installed version. However, _**lxml==4.3.4**_ does not satisfy _**==4.2.1**_.

### Dependency tree-----
```
kindred<version range:>
| +-bioc<version range:==1.2.3>
| | +-docutils<version range:==0.14>
| | +-lxml<version range:==4.2.1>
| +-docstringtest<version range:>
| +-fasteners<version range:>
| +-future<version range:>
| +-intervaltree<version range:>
| | +-sortedcontainers<version range:>=2.0,<3.0>
| +-lxml<version range:>
| +-networkx<version range:>
| +-numpy<version range:>
| +-pytest-socket<version range:>
| | +-pytest<version range:>=3.6.3>
| +-scikit-learn<version range:>
| +-scipy<version range:>
| +-six<version range:>
| +-spacy<version range:>
| +-sphinx<version range:==1.5.5>
| +-sphinx-rtd-theme<version range:>```
```

Thanks for your help.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Solution
1. Fix your direct dependency to be lxml==4.2.1. I have checked this revision will not affect your downstream projects now.

2. Ask your upstream project bioc to loose the version range of lxml. 

Which solution do you prefer, 1 or 2?

Please let me know your choice. I can submit a PR to solve this issue.

**Comment by jakelever** [comment]
Hi, sorry for the delay. And thanks for figuring this out. I've changed the dependencies around a bit to use a newer version of BioC and no longer requiring lxml directly. Hopefully this resolves this issue for you.
